ginga/cmap.py

```diff
@@ -13309,27 +13305,23 @@ def add_matplotlib_cmap(cm, name=None):
 def add_matplotlib_cmaps(fail_on_import_error=True):
     """Add all matplotlib colormaps."""
     try:
-        import matplotlib.pyplot as plt
-        from matplotlib import cm as _cm
-        if MPL_LT_3_4:
-            from matplotlib.cbook import mplDeprecation as MatplotlibDeprecationWarning
-        else:
-            from matplotlib import MatplotlibDeprecationWarning
+        from matplotlib import colormaps as _mpl_cm
+        from matplotlib import MatplotlibDeprecationWarning
     except ImportError:
         if fail_on_import_error:
             raise
         # silently fail
         return
 
     # NOTE: Update if matplotlib has new public API for this.
-    for name in plt.colormaps():
+    for name in _mpl_cm:
         if not isinstance(name, str):
             continue
         try:
             # Do not load deprecated colormaps
             with warnings.catch_warnings():
                 warnings.simplefilter('error', MatplotlibDeprecationWarning)
```

Comment:
I think the deprecation warning catching should be removed. It should no longer happen.

Reply:
@pllim, because those particular color maps have been removed some time ago? I guess we can add the code back if they deprecate more color maps in the future. Wondering why they deprecated color maps...

Reply:
@ejeschke , I don't follow matplotlib development closely, so not sure. But I do not see any warning now using the matplotlib I have locally. Do you?

I guess does not hurt to keep it as well.